tests/D2L.CodeStyle.Analyzers.Test/SpecRunner.cs

```diff
@@ -338,7 +338,10 @@ ImmutableArray<string> arguments
 
 					yield return new DiagnosticExpectation(
 						name: name,
-						arguments: arguments.Split( ',' ).Select( RemoveSingleLeadingSpace ).ToImmutableArray()
+						arguments: Regex.Split( arguments, @"(?<!\\\\)," )
+							.Select( arg => arg.Replace( "\\\\", string.Empty ) )

```

Comment:
```suggestion
						arguments: Regex.Split( arguments, @"(?<!\\\\)," )
							.Select( arg => arg.Replace( "\\\\,", "," ) )
```

Reply:
Is the reason for this just to make it only catch escaped commas?

Reply:
Right